app/services/analytics_events.rb

```diff
@@ -3538,6 +3538,11 @@ def show_password_button_clicked(path:, **extra)
     track_event('Show Password Button Clicked', path: path, **extra)
   end
 
+  # Tracks if a user clicks the "You will also need" accordion on the homepage
+  def sign_in_idv_requirements_accordion_clicked
+    track_event('IdV requirements accordion clicked')
```

Comment:
Should we prefix this with "Sign In" for consistency with how the method's named / how the frontend logging is called?

```suggestion
    track_event('Sign In: IdV requirements accordion clicked')
```